nautobot/ipam/filters.py

```diff
@@ -236,12 +205,20 @@ class PrefixFilterSet(
         field_name="vlan__vid",
         label="VLAN number (1-4095)",
     )
+    rir = NaturalKeyOrPKMultipleChoiceFilter(
```

Comment:
Our upgrading guide currently doesn't list *net-new* filter fields. Wonder if it should? Those are of course less impactful than the renamed/removed/changed filter fields when upgrading, but maybe we should include for completeness?